lib/pbench/test/unit/server/test_indexer.py

```diff
@@ -1,4 +1,7 @@
-from pbench.server.indexer import ResultData
+import pytest
+
+import pbench.server.indexer
+from pbench.server.indexer import init_indexing, ResultData
```

Comment:
We have to be able to patch some kind of object used by the target module.  So we can't import the class here and patch it because the other module uses it there.  If you can find way to do it, would love to know.